src/Adapter/Profile/QueryHandler/GetProfileForEditingHandler.php

```diff
@@ -68,4 +88,25 @@ private function getProfile(ProfileId $profileId)
 
         return $profile;
     }
+
+    /**
+     * @param int $imageId
+     *
+     * @return array|null
+     */
+    private function getAvatarUrl(int $imageId): ?array
+    {
+        $imagePath = _PS_PROFILE_IMG_DIR_ . $imageId . '.jpg';
```

Comment:
put _PS_PROFILE_IMG_DIR_ in the constructoir please

Reply:
Done